README.md

```diff
@@ -0,0 +1,52 @@
+
+# Hello Microverse
+
+This is project is going to help us to master github flow and get used with linters for checking errors in our css ans Html files.
+
+## Materials required for validation
+
+- npm install --save-dev hint@6.x
+- npx hint .
+- npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x       stylelint-csstree-validator@1.x
+
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Get Started
+To get a local copy up and running follow these simple example steps.
+
+Use this command to have the copy of the project in your Current directory
+-  git clone git@github.com:moise10r/Hello_World.git.
+
```

Comment:
- [ ] **OPTIONAL**
Please notice that in this CLI you've used the [SSH key](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/about-ssh), not the HTTPS link, and for this command line to work properly it should use the HTTPS link as defined below.
A nice tip to know when you're typing command lines or code snippets in a markup language is to put it inside `` to be like
`git clone https://github.com/moise10r/Hello_Microverse.git`